syntax = "proto3";

package syft.core.node.common.service;

import "proto/core/common/common_object.proto";
import "proto/core/io/address.proto";

message ExceptionMessage {
  syft.core.io.Address address = 1;
  syft.core.common.UID msg_id_causing_exception = 2;
  string exception_type = 3;
  string exception_msg = 4;
}
